pinc/genres.inc

```diff
@@ -1,92 +1,95 @@
 <?php
 
-$GENRES = array(
```

Comment:
$GENRES is also used in a noncvs file projects_with_odd_values.php. Should we update it?

Reply:
Good eye. Yes, it will need to get updated too. Generally I make a note for myself to do those edits on TEST when the change is merged into master and then into PROD when it gets published.